ui/components/Logo.tsx

```diff
@@ -41,8 +41,9 @@ export default styled(Logo)`
     width: auto;
     height: 32px;
   }
-  padding-left: 20px;
+  padding-left: 12px;
+  height: 65px;
   //nav width: 200px - space btwn nav and content: 24px - content padding: 24px. All together 248px - 20 for left padding to line up with detail page titles.
```

Comment:
I care less about the alignment of the breadcrumbs, more about the logo padding, but I think this comment should be removed since it's not relevant anymore